### ISSUE TITLE
TypeError: int() got a None

### ISSUE THREAD

**Issue by fpoli** [viperproject/bitbucket-issue-migration#29]
```
[2020-02-17 15:26:04] === Migrate isues and pull requests of bitbucket repository 'viperproject/viperserver' to github...
Load mapping of mercurial commits to git...
checking uniqueness of hg and git hashes...
check done
Get all bitbucket issues...
Get all 11 detailed bitbucket pull requests...
10/11...
Migrate bitbucket attachments to github...
Prepare github issues...
Prepare github issue #1 from bitbucket pull request...
Prepare github issue #2 from bitbucket pull request...
Prepare github issue #3 from bitbucket pull request...
Prepare github issue #4 from bitbucket pull request...
Traceback (most recent call last):
  File "./migrate-discussions.py", line 953, in <module>
    main()
  File "./migrate-discussions.py", line 949, in main
    bitbucket_to_github(bexport=bexport, gimport=gimport, cmap=cmap, args=args)
  File "./migrate-discussions.py", line 779, in bitbucket_to_github
    gissue_or_gpull = construct_gissue_or_gpull_from_bpull(bpull, bexport, cmap, args)
  File "./migrate-discussions.py", line 644, in construct_gissue_or_gpull_from_bpull
    issue_body = construct_gpull_request_body(bpull, bexport, cmap, args)
  File "./migrate-discussions.py", line 451, in construct_gpull_request_body
    sb.append(map_content(bpull["description"], cmap, args))
  File "./migrate-discussions.py", line 253, in map_content
    tmp = replace_implicit_links_to_prs(tmp, args)
  File "./migrate-discussions.py", line 109, in replace_implicit_links_to_prs
    return IMPLICIT_PR_LINK_RE.sub(replace_pr_link, body)
  File "./migrate-discussions.py", line 86, in replace_pr_link
    bpr_nr = int(match.group(2))
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

**Comment by fpoli** [comment]
The original PR should be this one: https://bitbucket.org/viperproject/viperserver/pull-requests/4/http-server/diff

**Comment by ArquintL** [comment]
fixed in 49a5a39
